index.html

```diff
@@ -11,35 +11,41 @@
       href = "css/main.css" />
     <link href="https://fonts.googleapis.com/css?family=Alfa+Slab+One"
       rel="stylesheet">
-
   </head>
 
   <body>
     <nav>
-      <div class="name">Breyana Scales</div>
-      <ul>
-        <li><a href="pages/contact.html">Contact</a></li>
-        <li><a href="pages/projects.html">Projects</a></li>
-        <li><a href="pages/cv.html">CV</a></li>
-        <li><a>About</a></li>
+      <div class="name">Breyana</div>
```

Comment:
I wonder if you might be interested in templating here-- your nav bar is repeated over different HTML files. I might totally be wrong about what templating is, but I think it means that you can define a feature/element once and then use it in different files. http://blog.teamtreehouse.com/creating-reusable-markup-with-the-html-template-element